presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -280,20 +293,24 @@ public Environment build(EnvironmentListener listener)
 
         private Environment build(Optional<EnvironmentListener> listener)
         {
-            // write directly to System.out, bypassing logging & io.airlift.log.Logging#rewireStdStreams
-            PrintStream out;
-            try {
-                //noinspection resource
-                out = new PrintStream(new FileOutputStream(FileDescriptor.out), true, Charset.defaultCharset().name());
-            }
-            catch (UnsupportedEncodingException e) {
-                throw new RuntimeException(e);
+            switch (outputMode) {
+                case DISCARD:
+                    log.warn("Containers logs are not printed to stdout");
+                    setContainerOutputConsumer(this::discardContainerLogs);
+                    break;
+
+                case PRINT:
```

Comment:
Can we print to stout also in write mode

Reply:
I've added PRINT_WRITE mode 👍🏻